feature-libs/cart/main/root/facade/active-cart.facade.ts

```diff
@@ -122,11 +122,7 @@ export abstract class ActiveCartFacade {
    */
   abstract addEntries(cartEntries: OrderEntry[]): void;
 
-  abstract requireLoadedCart(
-    customCartSelector$?: Observable<
-      StateUtils.ProcessesLoaderState<Cart | undefined>
-    >
-  ): Observable<StateUtils.ProcessesLoaderState<Cart | undefined>>;
```

Comment:
Didn't UI components need the state to display spinners ans react to errors?

Reply:
`requireLoadedCart` means require `loaded` cart. There is no case that we use it directly. Normally, we call it when we want to do something (e.g. add entry to cart), which needs a cart be ready.